src/pydata_sphinx_theme/assets/styles/_api.scss

```diff
@@ -87,3 +87,30 @@ table.field-list {
 .sig-name {
   color: rgba(var(--pst-color-inline-code), 1);
 }
+
+// change target color for dark theme
+dt:target {
+  background-color: rgba(var(--pst-color-target), 1);
+}
+
+// adapt ethical add to the theme
```

Comment:
Can you put this in a folder like `styles/components/ethical-ads.scss` and import it?

Reply:
same comments as for the `.html`file. The `components` does not yet exist, I think it would make more sense to simply create an ethical-ads.scss file and then you'll create the `components`folder when resolving #577.

Let me know what I should do

Reply:
Either one is fine with me - happy to just keep it as is